test/Altinn.Notifications.Tests/Notifications.Integrations/AuthorizationServiceTests.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System.Collections.Generic;
```

Comment:
could we move this into an Auuthorization folder to match existing structure for tests?
Can also consider moving the Testdata folder to the Notification.Integrations folder to keep it close to the tests that use them